adhocracy4/comments_async/templatetags/react_comments_async_with_categories.py

```diff
@@ -14,7 +14,7 @@
 
 
 @register.simple_tag(takes_context=True)
-def react_comments_async_with_categories(context, obj):
+def react_comments_async(context, obj, debate_app=True):
     request = context['request']
     user = request.user
```

Comment:
@fuzzylogic2000 is that what you meant?

Reply:
Yeah, but you can also add the other bit where you use react_comments_async in react_comments_async_with_categories, so that that still works on a+ without any further  changes